setupDB.py

```diff
@@ -0,0 +1,54 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import Column, Integer, Float, Date, String, VARCHAR
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine
+import pandas as pd
+
+# app = Flask(__name__)
+# app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:////tmp/test.db'
+# db = SQLAlchemy(app)
+
+Base = declarative_base()
+
+
+class ConferenceDB(Base):
+    __tablename__ = 'womenInTechResources'
+    #__table_args__ = {'sqlite_autoincrement': True}
+
+    id = Column(Integer, primary_key=True, nullable=False)
+    name = Column(VARCHAR(40))
```

Comment:
For `name=Column(VARCHAR(40))` , what does 40 stands for? how is it different from `Column(VARCHAR)` in below? I am new to db so just out of curiosity:)

Reply:
believe it tells you the max length of characters (in this case 40) you can store in that column